app/src/main/java/com/vero/tasky/agenda/data/local/entities/DeletedAgendaItemEntity.kt

```diff
@@ -0,0 +1,12 @@
+package com.vero.tasky.agenda.data.local.entities
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.vero.tasky.agenda.domain.model.AgendaItemType
+
+@Entity(tableName = "deleted_item")
+data class DeletedAgendaItemEntity(
```

Comment:
What if you want to also save the Updated/Created item that was updated/created during offline mode?